pkg/controllers/clusterresourceplacement/resource_selector.go

```diff
@@ -287,9 +287,9 @@ func (r *Reconciler) fetchAllResourcesInOneNamespace(namespaceName string, place
 	return resources, nil
 }
 
-// shouldSelectResource returns whether a resource should be propagated
+// shouldSelectResource returns whether a resource should be selected for propagation.
 func (r *Reconciler) shouldSelectResource(gvr schema.GroupVersionResource) bool {
-	if r.DisabledResourceConfig == nil {
+	if r.ResourceConfig == nil {
```

Comment:
```suggestion
       // By default, all of the APIs are allowed.
	if r.ResourceConfig == nil {
```

nit, add a comment to improve the readability